fe1-web/src/core/network/JsonRpcApi.ts

```diff
@@ -83,6 +84,28 @@ export async function subscribe(
   // propagate the catch() with the full error message, as it needs to be handled on a higher level
 }
 
+/**
+ * Unsubscribe from a channel
+ *
+ * @param channel - The channel from which we want to unsubscribe
+ * @param connections - An optional list of network connection if the message should only be sent on a subset of connections
+ */
+export async function unsubscribe(
```

Comment:
thanks for finally implementing this. I thought it'd been implemented a long time ago 😅